Build.props

```diff
@@ -0,0 +1,29 @@
+<Project>
```

Comment:
Please configure mandatory documentation but just for interfaces: https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/Configuration.md#documentation-requirements

Reply:
Added `stylecop.json` file with needed configuration.

Reply:
This should show up under SA1600, right? This seems that it doesn't run on build, just when you open a given file.

Reply:
Right, SA1600. Try to rebuild solution. I've got 233 warnings of SA1600.

Reply:
I tried that but they won't show up, not even if I switch the rule to Error. Does `dotnet build Finitive.sln --configuration Debug --no-incremental >build.log` output SA1600 violations for you? Not for me.

Reply:
Very strange. I also don't see SA1600 by using `dotnet build`. But I do have this warnings in Visual Studio.

Reply:
Actually, all SA16XX documentation rules were not present when using `dotnet build`. Adding `<GenerateDocumentationFile>true</GenerateDocumentationFile>` to `Build.props` seems that solved the problem. Please confirm that it is OK for you now.

Reply:
Works perfectly, thank you.